Sources/WormholySwift/Extensions/UIViewController+Ext.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Wormholy.swift
+//  Wormholy
+//
+//  Created by Paolo Musolino.
+//  Copyright © 2018 Wormholy. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+
+extension UIViewController {
+
+    static func currentViewController(_ viewController: UIViewController? = UIApplication.shared.windows.filter(\.isKeyWindow).first?.rootViewController) -> UIViewController? {
```

Comment:
I believe the use of `UIApplication.shared.windows.filter(\.isKeyWindow).first?.rootViewController` can be optimized by using `UIApplication.shared.keyWindow?.rootViewController` which is more efficient. Wdyt?

Reply:
@pmusolino [UIApplication.keyWindow](https://developer.apple.com/documentation/uikit/uiapplication/keywindow) is deprecated since iOS 13. It will give a compilation warning even though it continues working for now, so looks like it will not be much better.